packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -59,6 +59,7 @@
 InputID = OutputID = constr(regex=PROPERTY_KEY_RE)
 Inputs = Dict[InputID, InputTypes]
 Outputs = Dict[OutputID, OutputTypes]
+InputsUnits = Dict[InputID, str]
```

Comment:
``InputsUnits`` is not covering the case in which the input's value is an object

Say you have
```json
{
   "rho": 1,
   "theta": 45,
   "phi": 0
}
```
```json
{
   "rho":"mili-meter",
   "theta":"degrees",
   # phi  will use default
}
```